src/main/java/dev/symphony/harmony/mixin/food/ProportionalFoodTime.java

```diff
@@ -0,0 +1,29 @@
+package dev.symphony.harmony.mixin.food;
+
+import dev.symphony.harmony.config.HarmonyConfig;
+import net.minecraft.component.type.FoodComponent;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+// AUTHOR michiel1106
+// FEATURE change how long it takes to eat fooditems based on how much nutrition it gives
```

Comment:
Please, try to keep the credits/feature description comments consistent with the rest of the mod. Check any other feature as reference.

Reply:
LGTM 👍